ForgeRock Cloud APIs.postman_collection.json

```diff
@@ -4050,7 +4055,7 @@
 									"raw": ""
 								},
 								"url": {
-									"raw": "{{tenantApiV1Url}}/password-policy",
+									"raw": "{{tenantApiV1Url}}/password-policy?=",
```

Comment:
Does the password policy require the ?= query in the endpoint?

Reply:
Good catch. It works fine without. Removing.